datafusion/sql/src/unparser/plan.rs

```diff
@@ -800,6 +799,7 @@ impl Unparser<'_> {
     fn unparse_table_scan_pushdown(
         plan: &LogicalPlan,
         alias: Option<TableReference>,
+        select: &mut SelectBuilder,
```

Comment:
This seems to only use the `select.already_projected()` method - perhaps this could just be a bool instead of passing the whole struct reference?

Reply:
Updated - thank you!